src/services/EventsExpressService.js

```diff
@@ -22,7 +22,7 @@ export default class EventsExpressService {
   };
 
   getPhoto = async Url => {
```

Comment:
Url must be lower case.
Write `url` instead `Url` in full file and rename some vars (example: url -> callUrl).